UPGRADE-3.4.md

```diff
@@ -0,0 +1,10 @@
+UPGRADE FROM 3.3 to 3.4
+=======================
+
+FrameworkBundle
+---------------
+
+ * Using `KERNEL_DIR` or the automatic guessing based on the `phpunit.xml` file
+   location is deprecated since 3.4. Either set `KERNEL_CLASS` in your `phpunit.xml` 
+   to the fully-qualified class name of your Kernel or override the `KernelTestCase::createKernel()` 
+   method. Not setting `KERNEL_CLASS` will throw an exception on 4.0.
```

Comment:
Yes indeed. I mean not setting `KERNEL_CLASS` when using the `KernelTestCase::createKernel()` base implementation. Of course if you overwrite it your on your own (and it's already suggested).
I don't know how to rephrase it better if the current version is not enough. I'll take any other suggestion.